lib/ReactViews/Preview/Description.jsx

```diff
@@ -208,25 +190,20 @@ const Description = observer(
               </Choose>
             </If>
 
-            <If
-              condition={
-                catalogItem.dataUrlType &&
-                catalogItem.dataUrlType !== "none" &&
-                catalogItem.dataUrl
-              }
-            >
+            <If condition={dataUrls && dataUrls.length > 0}>
               <h4 className={Styles.h4}>{t("description.dataUrl")}</h4>
-              <p>
+              <For each="dataUrl" index="i" of={dataUrls}>
                 <Choose>
                   <When
                     condition={
-                      catalogItem.dataUrlType.indexOf("wfs") === 0 ||
-                      catalogItem.dataUrlType.indexOf("wcs") === 0
+                      dataUrl.type?.indexOf("wfs") === 0 ||
```

Comment:
could use String.prototype.includes here instead of indexOf. It's a nitpick so feel free to ignore

Reply:
Yep i agree, the new functions are nicer than indexOf - I've used `startsWith` instead